plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -24,6 +21,36 @@
 
 catalog = i18nCatalog("cura")
 
+class SliceInfoJob(Job):
+    data = None
+    url = None
+
+    def __init__(self, url, data):
+        super().__init__()
+        self.url = url
+        self.data = data
+
+    def run(self):
+        if not self.url or not self.data:
+            Logger.log("e", "URL or DATA for sending slice info was not set!")
+            return
+
+        # Submit data
+        kwoptions = {"data" : self.data,
+                     "timeout" : 5
+                     }
+
+        if Platform.isOSX():
+            kwoptions["context"] = ssl._create_unverified_context()
+
+        try:
+            f = urllib.request.urlopen(self.url, **kwoptions)
+            Logger.log("i", "Sent anonymous slice info to %s", self.url)
+            f.close()
+        except urllib.error.HTTPError as http_exception:
+            Logger.log("e", "An exception occurred while trying to send slice information: %s" %(repr(http_exception)))
+        except Exception: # Includes urllib.error.HTTPError, was discussed to be handled like any other exception 
+            Logger.logException("e", "An exception occurred while trying to send slice information")
```

Comment:
We don't have "rules" to decide when to use logException() vs just a log() that includes the exception text, but I would only use logException in cases where the context/traceback might be unknown (ie from very generic/often used classes such as SettingFunction, Validator, ContainerRegistry, which gets called from many different classes/cases). 
In this case it just add unnecessary log verbosity. But since it was already used here, I'll let it slide.